scripts/skydb.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Demo script that uploads data passed in as CLI arguments.
+ *
+ * Example usage: node scripts/skydb.js <dbVersion> <dataKey>
+ *
+ * Example for Skydb v1 usage: node scripts/skydb.js -v1 testKey
+ *
+ * Example for Skydb v2 usage: node scripts/skydb.js -v2 testKey
+ *
+ */
+
+const process = require("process");
+
+const { SkynetClient, genKeyPairFromSeed } = require("..");
+const { publicKey, privateKey } = genKeyPairFromSeed("Super Save Test Secret");
```

Comment:
I think it would be cool to print out the keys, that way the user can get to his data again. Also cool if you could (optionally) specify the private key.